deployments/i2b2.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.9'
+services:
+  i2b2:
+    image: ${I2B2_DOCKER_IMAGE:?}
+    build:
+      context: ../build/package/i2b2
+    ports:
+      - "8080:8080"
```

Comment:
If keycloak is also deployed on 8080, indeed there will be a conflict. At the moment only postgresql is started from the geco deployment though so there is no immediate issue. I am updating this to prevent issue in the future.